docker-compose.yml

```diff
@@ -47,6 +47,16 @@ services:
             - RABBITMQ_URL=amqp://guest:guest@event-bus:5672
         depends_on:
           - article-store-postgres
+    article-store-exchange-setup:
+        build:
+            context: ./article-store
+        volumes:
+            - ./article-store:/srv/app
+        environment:
+            - RABBITMQ_URL=amqp://guest:guest@event-bus:5672
+        command: pipenv run python manage.py setup_exchanges
```

Comment:
doesn't block the other containers from progressing while these are declared, but they don't need to as they idempotently declare them as well just in case